### ISSUE TITLE
React Native

### ISSUE THREAD

**Issue by oliv3** [traccar/traccar-client-android#395]
Are there any plans to build a client with [React Native](https://facebook.github.io/react-native/) ?

This way there would be only one code base for both iOS and Android.

**Comment by ivan-avalos** [comment]
I think that would be a great idea, at least making it more reactive. But never expect that to change, that's the way the Traccar developers prefer and upgrading technologies would make it unnecessarily complex for them.

**Comment by ivan-avalos** [comment]
The only solution here would be creating your own unofficial reactive **Traccar** application, because you know software engineers, they won't move a finger if the stuff "works", but of course that depends on many things, I'm not saying that all the software developers are like that.

**Comment by htetwaiyan** [comment]
I already developed and used  in my flutter app

**Comment by JEuler** [comment]
> I already developed and used in my flutter app ,
> realtime socket page and track all car.

@htetwaiyan So, could you share the code? :)

**Comment by htetwaiyan** [comment]
I can't. Because I sell this app to traccar user with customize ui.  If you
buy I will pay full source code

**Comment by Andyeyo** [comment]
> > I already developed and used in my flutter app ,
> > realtime socket page and track all car.
> 
> @htetwaiyan So, could you share the code? :)

@JEuler  I post code as how to use flutter with traccar (web socket). Please check https://stackoverflow.com/a/61684225/10729436

**Comment by JEuler** [comment]
@Andyeyo Cool. I was looking about a year ago, there were nothing like SDK, cool 👍

**Comment by Andyeyo** [comment]
> @Andyeyo Cool. I was looking about a year ago, there were nothing like SDK, cool

Don't worry @JEuler if you need help, let me know. Regards

**Comment by nucl123** [comment]
> I already developed and used in my flutter app ,
> realtime socket page and track all car.

Interested in purchasing it, please share the details on how to purchase it. thanks

**Comment by bonaggioandroid** [comment]
> > @Andyeyo Cool. I was looking about a year ago, there were nothing like SDK, cool
> 
> Don't worry @JEuler if you need help, let me know. Regards

Andre consegue me ajudar? please my help traccar client

**Comment by bonaggioandroid** [comment]
> > I already developed and used in my flutter app ,
> > realtime socket page and track all car.
> 
> Interested in purchasing it, please share the details on how to purchase it. thanks

price ?

**Comment by jcardus** [comment]
I have a fork of traccar-web with one codebase for ios and android, using capacitor.

**Comment by tananaev** [comment]
Capacitor makes a WebView app, which is what our official apps are essentially. This discussion is about native apps.

**Comment by jcardus** [comment]
Yes I know, the discussion is about native apps and having one code base, I'm just talking about the latter. There are two projects for the official apps.
